src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/XUnit/XUnitTestAssemblyFinishedCtorIntegration.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="XUnitTestAssemblyFinishedCtorIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit
+{
+    /// <summary>
+    /// Xunit.Sdk.TestAssemblyRunner`1.RunAsync calltarget instrumentation
```

Comment:
nit: wrong name

Reply:
Ups, I've already merge it, I'll create a PR to fix this.

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/1541